mocha.opts

```diff
@@ -1,2 +1,2 @@
---recursive packages/**/test/**/*.test.js
+--recursive packages/*/test/**/*.test.js
```

Comment:
You can also add `--exclude **/node_modules/**` for this.

Reply:
Ah, yes, that was the problem I was having. Probably makes more sense to do this explicitly.